src/main/java/com/b44t/messenger/DcContext.java

```diff
@@ -236,6 +236,19 @@ public boolean isChatmail() {
       return getConfigInt("is_chatmail") == 1;
     }
 
+    // Called for new profiles on chatmail servers that are "single device" initially;
+    // to save server disk space, we make use of that delete all messages immediately after download.
+    public void assumeSingleDevice() {
```

Comment:
I am wondering if this isn't better handled automatically by core, similar to how the defaults are selected for some providers via provider-db, core could already have the correct default value for chatmail instances

I think this will set wrong defaults for mailcow deployments with the QR-invites

Reply:
well, we want to have an update soon, and the issue is mainly about android, as user onboard there mostly. if things work out, we can think over doing that in core.

Reply:
> @adbenitez i am afk - can you check what would be a good place to call assumeSingleDevice()?

nm, i shifted the call and check for isChatmail() before changing the autodel settings.

still, makes sense if you have an additional look if the last commit makes sense to you :)